devops/actions/llvm_test_suite/action.yml

```diff
@@ -0,0 +1,105 @@
+name: 'Run llvm-test-suite testing'
+description: 'Run E2E SYCL tests from llvm-test-suite'
+inputs:
+  test_ref:
+    description: 'commit-ish identifier for test repo'
+    required: true
+    default: 'intel'
+  sycl_artifact:
+    description: 'Name of the artifact, that contains compiler toolchain'
+    required: true
+  sycl_archive:
+    description: 'Name of SYCL toolchain archive file'
+    required: false
+    default: 'llvm_sycl.tar.xz'
+  lit_artifact:
+    description: 'Name of the artifact, that contains LIT tool'
+    required: true
+  lit_archive:
+    description: 'Name of LIT archive file'
+    required: false
+    default: 'lit.tar.xz'
+  results_name_suffix:
+    description: 'Name suffix of the results artifact'
+    required: true
+  check_sycl_all:
+    description: 'List of targets to include in testing'
```

Comment:
I read `targets` as CMake targets and was really confused by parameter name choice `check_sycl_all`. 

Here is how this parameters is described in the documentation:

> CHECK_SYCL_ALL - defines selection of multiple SYCL backends with set of target devices per each to be tested iteratively. Value is semicolon-separated list of configurations. Each configuration includes backend separated from comma-separated list of target devices with colon. Example:
> 
> -DCHECK_SYCL_ALL="opencl:cpu,host;level_zero:gpu,host;cuda:gpu;hip:gpu"

It uses "backends" and "devices". I think "List of target devices to test" should sound less ambiguous.

Reply:
Fixed